bVNC/src/main/res/values/strings.xml

```diff
@@ -33,6 +33,7 @@
     <string name="connection_setup_user">User@Domain</string>
     <string name="connection_setup_password">Password (saved if entered)</string>
     <string name="enter_password">Please enter password</string>
+    <string name="enter_password_auth_failed">Authentication failed. Please re-enter your credentials.</string>
```

Comment:
Please pull in latest from master get Spanish translations in, and then add the new english strings as-is to all languages.